nncf/experimental/openvino_native/hardware/fused_patterns.py

```diff
@@ -715,6 +715,44 @@ def create_linear_biased_activation_elementwise():
     return linear_biased
 
 
+@OPENVINO_HW_FUSED_PATTERNS.register(PatternNames.MATMUL_SOFTMAX_MATMUL)
+def create_matmul_softmax_matmul():
```

Comment:
Do we need to keep this pattern as `model_type`-specific? 
cc @l-bat, @alexsu52

Reply:
IN POT there are no model_type specific patterns as far as I know